src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -66,7 +66,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
 
         if(invalidFields.length > 0){
             let fieldListAsString = invalidFields.join(', ');
-            showToast(this, 'Invalid Form', 'The following fields are required: ' + fieldListAsString, 'error');
+            showToast('Invalid Form', 'The following fields are required: ' + fieldListAsString, 'error');
```

Comment:
This isn't part of any particular work item, but the required fields message was not displaying correctly.